src/styles/base/_override.typography.scss

```diff
@@ -15,123 +15,113 @@
 .ds-text-heading--lg,
 .ds-text-heading--md,
 .ds-text-heading--sm {
+  font-family: Montserrat;
   font-weight: $font-semibold;
-  font-family: "Montserrat";
-}
-
-.ds-title,
-.ds-text-heading--4xl {
-  font-weight: 300;
 }
 
 .ds-h1,
 .ds-text-heading--3xl {
-  font-size: $h1-mobile-font-size;
-  line-height: 32px;
-
+  font-size: $font-size-2xl;
+  line-height: 40px;
+  margin-bottom: 44px;
   @media screen and (min-width: $width-md) {
-    line-height: 47px;
     font-size: $font-size-3xl;
+    line-height: 48px;
+    margin-bottom: 52px;
   }
 }
 
 .ds-h2,
 .ds-text-heading--2xl {
-  margin-top: 36px;
-  margin-bottom: 12px;
+  font-size: $font-size-2xl;
+  line-height: 40px;
```

Comment:
I think you could use `$spacer-5` here from [the core layout variables ](https://github.com/CMSgov/design-system/blob/development/packages/design-system/src/styles/settings/variables/_layout.scss#L18)